### DISCUSSION TITLE
some comments on the model of oscillations in yeast glycolysis

### DISCUSSION THREAD

**Discussion by JuntaoHuang** [DENG-MIT/CRNN#3] [Ideas]
The model of oscillations in yeast glycolysis is an interesting application!

The ODE system is in rational form, but it is the reduced model of the ODE system in algebraic form. This reminds me of the Koopman operator. The ODE system in a complex formulation will be simplified to linear form if mapping into high dimensions.

**Comment by jiweiqi** [comment]
Yep, it is great to connect it to the Koopman operator. I am not familiar with the Koopman operator although I have tried to understand...
Would be great to formulate the problem in the form of a Koopman operator which might introduce more benefits and potentials.
